### PR TITLE
{bio}[foss/2020b] Octopus-vcf v0.7.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Are you going with "octopus" to avoid naming conflict with archived Octopus? Because the naming conflict should be case insensitive ($EBROOTOCTOPUS and would conflict for example)

**Comment by migueldiascosta** [comment]
also, it's not only about the archived one, more recent easyconfigs for the Octopus DFT code (https://github.com/easybuilders/easybuild-easyconfigs/pulls?q=is%3Apr+Octopus+) haven't been merged because it's hard to get the testsuite to pass for all architectures, but we should definitely come back to that...

**Comment by Micket** [comment]
I would suggest that probably both of these PRs should just use  a more extended name; "Octopus-DFT" and, uh.. "Octopus-whatever-the-heck-this-is"

**Comment by verdurin** [comment]
Yes, I was aware of the other ones but forgot that we need to be case-insensitive. Added `-vcf` given it's a variant caller, and now using `Octopus` as per upstream.

File: easybuild/easyconfigs/o/Octopus-vcf/Octopus-vcf-0.7.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['b544121cb132634a1156e79c0a2ff19abacd3fb4ad5b711dec9dc664b6fcf49a']
+
+dependencies = [
+    ('CMake', '3.18.4'),
+    ('Boost', '1.74.0'),
+    ('HTSlib', '1.11'),
+    ('GMP', '6.2.0'),
+    ('Python', '3.8.6'),
+    ('git', '2.28.0', '-nodocs'),
```
Comment (Micket) [inline easybuild/easyconfigs/o/Octopus-vcf/Octopus-vcf-0.7.1-foss-2020b.eb:26]: Does it really need git and CMake at runtime?

File: easybuild/easyconfigs/o/Octopus-vcf/Octopus-vcf-0.7.1-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ('HTSlib', '1.11'),
+    ('GMP', '6.2.0'),
+    ('Python', '3.8.6'),
+]
+
+configopts = '-DHTSlib_NO_SYSTEM_PATHS=TRUE -DHTSLIB_ROOT=$EBROOTHTSLIB '
+
+separate_build_dir = True
+
+preinstallopts = 'if [ ! -d %(installdir)s/bin ]; then mkdir %(installdir)s/bin; fi'
```
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/o/Octopus-vcf/Octopus-vcf-0.7.1-foss-2020b.eb]: Yes, I think that must have lingered from an earlier iteration - took a while to find the right easyblock for this. Now removed.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node374.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/e37c87e807d291dadb45089742acf1b3 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**